client-v2/src/services/faciaApi.ts

```diff
@@ -166,7 +166,9 @@ async function saveClipboard(
   }
 }
 
-async function saveOpenFrontIds(frontIds?: string[]): Promise<void> {
+async function saveOpenFrontIds(frontIds?: {
+  [id: string]: string[];
```

Comment:
Does this `id` in `id: string` represent priority? If so can we rename it?

Reply:
Good point, yes -- this could do with a rename.